widgetssdk/src/main/java/com/glia/widgets/view/floatingvisitorvideoview/FloatingVisitorVideoCoordinatorLayout.java

```diff
@@ -85,12 +85,12 @@ public void onDestroy() {
 
     @Override
     public void showOnHold() {
-        // TODO- will be implemented in next task
+        post(() -> floatingVisitorVideoContainer.showOnHold());
```

Comment:
could you use rxjava or java streams to switch to ui thread somewhere earlier?

Reply:
Will do changes when buttons order task comes in 
https://glia.atlassian.net/browse/MUIC-718